hapi-fees/pom.xml

```diff
@@ -20,7 +20,7 @@
   <dependencies>
     <dependency>
       <groupId>com.hedera.hashgraph</groupId>
-      <artifactId>hedera-protobuf-java-api</artifactId>
+      <artifactId>hapi-utils</artifactId>
       <version>0.11.1-SNAPSHOT</version>
```

Comment:
Should the version be `0.12.x-SNAPSHOT` or do we want this changes to be in 0.11.x release?

Reply:
The version for the `hapi-utils` artifact will always match the parent version in the Services _pom.xml_ (just like for `hapi-fees`, `test-clients`, etc).

Reply:
I see. My bad. I thought the `hapi-protobuf` is already in `0.12.0`, so I assume `hedera-services` shall also be `0.12.0`.